13-spatial-cv.Rmd

```diff
@@ -18,9 +18,12 @@ library(mlr)
 - Required data will be downloaded in due course.
 
 ## Introduction
-In the beginning, we pointed out that there are several programming languages suitable for command-line based geocomputing (section \@ref(software-for-geocomputation)).
-The special advantage of doing geocomputation with R is combining geocomputing with R's unparalleled statistical power.
-In this chapter we will introduce predictive mapping by means of statistical learning [@james_introduction_2013] while using spatial cross-validation for a bias-reduced assessment of the model performance - something which is probably only easily doable with R at the moment.
+
+Section \@ref(software-for-geocomputation) mentioned several programming languages suitable for command-line based geocomputation.
+The advantages of geocomputation with R were discussed, including its unparalleled statistical power.
+This chapter makes use of some of this statistical power, by demonstrating methods for predictive mapping by means of statistical learning [@james_introduction_2013].
+The focus is the use of spatial cross-validation (or 'spatial CV' for short, a term we will define shortly) to assess model performance and reduce spatial bias.
```

Comment:
Will likely be a new term for most readers so we should point them to a definition, e.g.:

```
a term defined in \@ref(spatial-crossvalidation)
```

Slightly unusual to have section named the same as the chapter it's in so may be worth thinking about another name for that section.

Reply:
Thanks! Chapter and section naming is still work in progress, hence, the current names are more or less just placeholders.